source/NVDAHelper.py

```diff
@@ -481,9 +483,9 @@ def nvdaControllerInternal_openConfigDirectory():
 	return 0
 
 
-class RemoteLoader64(object):
+class RemoteLoader(object):

```

Comment:
This appears to be an API breakage?

Should we retain an alias?

```py
class RemoteLoader64(RemoteLoader):
	def __init__(self):
		loaderDir = ...
		super().__init__(loaderDir)
```

Reply:
If we want to deprecate this variable, I suggest marking this as private, and also removing the no longer relevant `object` superclass.

```suggestion
class _RemoteLoader:
```

Reply:
RemoteLoader is now private.
Re breaking change: I've mentioned this in changes.t2t under breaking changes. 
I cannot think of any valid (or safe) past external usage of RemoteLoader64. This always should have been (and was somewhat expected that) this was already private. Thus I do not believe adding a backwards compat class is useful.